locales/de/messages.json

```diff
@@ -18,8 +18,8 @@
     "Kind": "👦",
     "Weihnachtsbaum": "🎄",
     "Nahaufnahme": "👀",
-    "Wolke": "☁️",
-    "Wolken": "☁️",
+    "Wolke": "☁",
+    "Wolken": "☁",
```

Comment:
the characters had modifiers which were removed. If you want to keep them, please edit.